### DISCUSSION TITLE
Breach checkboxes

### DISCUSSION THREAD

**Discussion by SMG001** [bcholmes/StarTrek2d20#110] [General]
Hello! It's me again, I don't know if this is a bug or not, but for some reason, there doesn't seem to be any breach checkboxes on 24th Century sheets, and for some reason, while the 23rd Century sheets have some, they aren't really... complete. Like in a scale 4 ship there's only 2 breach checkboxes (When I say checkboxes, I mean the digital checkbox, not the actual ones printed on the sheet). Is this normal?

**Comment by bcholmes** [comment]
I think this should have been fixed with the last round of Starship sheet updates. (I *think* that all the sheets with a "Special Rules" section now have breach checkboxes). Let me know if that's not true.
